aptos-move/framework/aptos-framework/sources/Genesis.move

```diff
@@ -96,8 +97,12 @@ module AptosFramework::Genesis {
 
         TransactionPublishingOption::initialize(core_resource_account, initial_script_allow_list, is_open_module);
 
-        TestCoin::initialize(core_resource_account, 1000000);
-        TestCoin::mint_internal(core_resource_account, Signer::address_of(core_resource_account), 18446744073709551615);
+        // This is testnet-specific configuration and can be skipped for mainnet.
+        // Mainnet can call Coin::initialize<MainnetCoin> directly and give mint capability to the Staking module.
+        let burn_cap = TestCoin::initialize(&core_framework_account, core_resource_account);
+
+        // Give TransactionFee BurnCapability<TestCoin> so it can burn gas.
+        TransactionFee::store_test_coin_burn_cap(&core_framework_account, burn_cap);
```

Comment:
we don't want to store anything in the 0x1 address, we'll remove the account later as well.

Reply:
This is related to the discussion on friendship vs storing caps in system accounts. This PR is doing the latter approach, but either way we don't need to have signing control over the 0x1 address. I'm planning to remove the key rotation call for core_framework_account in a separate commit once I finish fixing all test failures.

Reply:
I think it is generally safer to assume that there will be some sort of governance hosted at 0x1 or some equally impossible to generate address.

Reply:
we already have one governance account, do we want a different one? it's probably better to separate from 0x1 too

Reply:
This is not exactly a governance account in the sense that we don't control the key for it. This is a special account where system capabilities are stored. I think it makes sense to use 0x1 since all of the system modules are deployed there and it makes sense that the capabilities needed for those modules to interact are also encapsulated there.

Reply:
I don't know if account separation matters so much, but it probably doesn't hurt to separate shared governance accounts based upon the role...

* Framework
* Staking / rewards
* Upgrades / versions

but that feels like a conversation that requires more thought